modules/conflict_resolver/help/conflict_resolver.md

```diff
@@ -1,9 +1,29 @@
 # Conflict Resolver
 
-This module is designed to identify and resolve data conflicts in a study. LORIS uses double data entry, which means you may be asked to enter critical data twice in duplicate instrument forms. The Conflict Resolver will flag any discrepancies between the two entries (called *unresolved conflicts*) and allow you to fix or "resolve" them easily. 
+The Conflict Resolver module enables studies to identify and resolve
+typos and discrepancies that arise between initial data entry and double
+data entry forms.
 
-Conflicts are displayed in a table with two separate tabs—resolved and unresolved conflicts. By default, the module displays the *Unresolved Conflicts* tab. You can narrow this list of results using any combination of filters in the *Selection Filters* section. 
+Unresolved and Resolved conflicts are displayed on two separate tabs of
+the module. Unresolved conflicts are displayed by default upon accessing
+the module. To access resolved conflicts, click on the “Resolved” tab.
 
-You can resolve all conflicts in the **Correct Answer** column of the resulting table. Click the drop-down to reveal two different responses—this is the conflict. These are two distinct values that were entered initially and during double data entry—they should be the same value. After cross-checking your data records for accuracy, select the correct response of the two. 
+Selection filters can be applied to search for a particular subject
```

Comment:
This help text's writing style doesn't seem to follow `docs/wiki/99_Developers/Help_Style_Guide.md`

In particular, it's changing it from the second person to the third person imperative

```
* Use second person point of view rather than third
  * e.g. “This module allows *you* to browse…” (rather than “This module allows *users* to browse…”)
```